index.html

```diff
@@ -185,5 +185,38 @@ <h1>Skills</h1>
                  </ul>
              </div>
           </section>
+          <footer class="footer-items">
+             <div class="feedback">
+              <h2 class="feedback-text">
+                  I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+              </h2>                    
+              <form action="https://formspree.io/f/xrgwgday" method="POST">
+                  <input class="form-input" type="text" id="fullname" name="fullname" placeholder="Full name" maxlength="30" required>
+                  <input class="form-input" type="email" id="email" name="email" placeholder="Email address" required>
+                  <textarea id="message" name="message" rows="5" placeholder="Write me something..." maxlength="500" required></textarea>                        
+                  <input class="submit-btn"  type="submit" value="Get in touch">
```

Comment:
- [ ] Well done 👏🏻 , you did an awesome job by implementing the changes as per the previous code reviewer's request 👍🏻 but there is one thing you missed and as per the previous code reviewer and what I noticed that it's important to also include styles for the hover and pressed states to match the design guidelines. Please ensure that you add appropriate styles to enhance the interactivity and visual feedback of the button when hovered or pressed. This will provide a more polished and cohesive user experience. Keep up the good work!

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Figma Design System

![Screenshot from 2023-07-14 12-50-35](https://github.com/Obimbo07/Portfolio-2/assets/66476329/a06fdf84-2b7a-44b9-8bb4-1f5a318637ed)

### Practical Example
[Screencast from 14-07-23 12:47:25 PM IST.webm](https://github.com/Obimbo07/Portfolio-2/assets/66476329/edcbb98c-d49f-49cb-a65e-547ced833fa6)